shared/src/main/java/com/couchbase/lite/DocumentReplicatedListener.java

```diff
@@ -0,0 +1,10 @@
+package com.couchbase.lite;
+
+public interface DocumentReplicatedListener {
```

Comment:
The correct name is `DocumentReplicationListener`.

Reply:
The class name hasn't been correct yet. Could you change it to DocumentReplicationListener?